frontend/src/static/js/components/webstatus-overview-cells.ts

```diff
@@ -23,9 +23,9 @@ import {
 import {BROWSER_ID_TO_ICON_NAME, FeatureSortOrderType} from '../api/client.js';
 import {ifDefined} from 'lit/directives/if-defined.js';
 import {
-  getTopCssIdentifierTemplate,
-  getTopHtmlIdentifierTemplate,
-} from './utils.js';
+  TOP_CSS_INTEROP_ISSUES,
+  TOP_HTML_INTEROP_ISSUES,
+} from '../utils/constants.js';
 
 const MISSING_VALUE = html``;
 
```

Comment:
I think you're missing the import for the badge

```suggestion
import './webstatus-feature-badge.js';
```